test-integration/replication.test.js

```diff
@@ -29,3 +32,13 @@ test.serial('All instances return uninitialized status error', async (t) => {
     }, { code: 94 })
   }
 })
+
+test.serial('Replication task can pick up uninitialized state', async (t) => {
```

Comment:
I would like to see a `unit` test for this same thing that also checks the inverse.